src/qtiCommonRenderer/renderers/interactions/AssociateInteraction.js

```diff
@@ -492,8 +492,9 @@ var render = function(interaction) {
                             onend: function(e) {
                                 var $target = $(e.target);
                                 $target.removeClass('dragged');
-                                _resetSelection();
-
+                                _.delay(function(){
+                                    _resetSelection();
+                                });
```

Comment:
The reason of placing delay here is that there was timing conflict between "draggable" and "drag-zone" elements.
The elder version of interactjs lib was handing them in such order that "dropzone" event was executed before draggable, and newer version handle events asynchronously despite of type of elements it wraps. 

So that solution allows to execute "reset selection" method od draggable element after "dropzone" events, without rewriting whole module.